code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -230,6 +182,104 @@
 				return TRUE
 	. = ..()
 
+/mob/living/carbon/human/proc/cpr(mob/living/carbon/human/H, starting = FALSE, cpr_mode)
```

Comment:
Please give this proc a more informative name, like `start_compressions` or something.